storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/operations/get/GetObjectStub.java

```diff
@@ -0,0 +1,174 @@
+
+    @NonNull
+    final GetResolver<TestItem> getResolver;
+
+    @NonNull
+    private final Cursor cursor;
+
+    @NonNull
+    final TestItem item;
+
+    @NonNull
+    private final SQLiteTypeMapping<TestItem> typeMapping;
+
+    private final boolean withTypeMapping;
+
+    @SuppressWarnings("unchecked")
+    private GetObjectStub(boolean withTypeMapping) {
+        this.withTypeMapping = withTypeMapping;
+
+        storIOSQLite = mock(StorIOSQLite.class);
+        internal = mock(StorIOSQLite.Internal.class);
+
+        when(storIOSQLite.internal())
+                .thenReturn(internal);
+
+        query = Query
+                .builder()
+                .table("test_table")
+                .build();
+
+        rawQuery = RawQuery
+                .builder()
+                .query("select * from who_cares")
+                .observesTables("test_table")
+                .build();
+
+        getResolver = mock(GetResolver.class);
+        cursor = mock(Cursor.class);
+
+        item = new TestItem();
+
+        when(cursor.getCount())
+                .thenReturn(1);
+
+        when(cursor.moveToNext()).thenAnswer(new Answer<Boolean>() {
+            int invocationsCount = 0;
+
+            @Override
+            public Boolean answer(InvocationOnMock invocation) throws Throwable {
+                return invocationsCount++ < 1;
+            }
+        });
+
+        when(storIOSQLite.get())
+                .thenReturn(new PreparedGet.Builder(storIOSQLite));
+
+        when(storIOSQLite.observeChangesInTables(eq(singleton(query.table()))))
+                .thenReturn(Observable.<Changes>empty());
+
+        assertThat(rawQuery.observesTables()).isNotNull();
+
+        when(storIOSQLite.observeChangesInTables(rawQuery.observesTables()))
+                .thenReturn(Observable.<Changes>empty());
+
+        when(getResolver.performGet(storIOSQLite, query))
+                .thenReturn(cursor);
+
+        when(getResolver.performGet(storIOSQLite, rawQuery))
+                .thenReturn(cursor);
+
+        when(getResolver.mapFromCursor(cursor))
+                .thenReturn(item);
+
+        typeMapping = mock(SQLiteTypeMapping.class);
+
+        if (withTypeMapping) {
+            when(internal.typeMapping(TestItem.class)).thenReturn(typeMapping);
+            when(typeMapping.getResolver()).thenReturn(getResolver);
+        }
+    }
+
+    @NonNull
+    static GetObjectStub newInstanceWithoutTypeMapping() {
+        return new GetObjectStub(false);
+    }
+
+    @NonNull
+    static GetObjectStub newInstanceWithTypeMapping() {
+        return new GetObjectStub(true);
+    }
+
+    void verifyQueryBehavior(@Nullable TestItem actualItem) {
+        // should be called once
+        verify(storIOSQLite).get();
+
+        // should be called only once
+        verify(getResolver).performGet(storIOSQLite, query);
+
+        // should be called only once
+        verify(getResolver, times(1)).mapFromCursor(cursor);
```

Comment:
`times(1)` is default behavior of `verify`, so it can be removed, again, sorry if you copied it from our code :)